File: libatsc3Module/src/main/java/org/ngbp/libatsc3/ndk/a331/BroadcastSvcSignaling.java
package org.ngbp.libatsc3.ndk.a331;

public class BroadcastSvcSignaling {
    int slsProtocol;
    int slsMajorProtocolVersion;
    int slsMinorProtocolVersion;
    String slsDestinationIpAddress;
    String slsDestinationUdpPort;
    String slsSourceIpAddress;

    public int getSlsProtocol() {
        return slsProtocol;
    }

    public int getSlsMajorProtocolVersion() {
        return slsMajorProtocolVersion;
    }

    public int getSlsMinorProtocolVersion() {
        return slsMinorProtocolVersion;
    }

    public String getSlsDestinationIpAddress() {
        return slsDestinationIpAddress;
    }

    public String getSlsDestinationUdpPort() {
        return slsDestinationUdpPort;
    }

    public String getSlsSourceIpAddress() {
        return slsSourceIpAddress;
    }
}

File: nextgen-middleware/src/main/java/com/nextgenbroadcast/mobile/middleware/atsc3/entities/service/BroadcastSvcSignaling.kt
package com.nextgenbroadcast.mobile.middleware.atsc3.entities.service

data class BroadcastSvcSignaling (
    var slsProtocol: Int = 0,
    var slsMajorProtocolVersion: Int = 0,
    var slsMinorProtocolVersion: Int = 0,
    var slsDestinationIpAddress: String? = null,
    var slsDestinationUdpPort: String? = null,
    var slsSourceIpAddress: String? = null
)